i18n/locales/en.ts

```diff
@@ -127,9 +127,9 @@ const translations = {
         'The message you found was encoded in a format called HEX. Now we will run a command to turn it into ASCII, which we will be able to read.',
       paragraph_two:
         'Copy and paste the command below into the Terminal to the right and press “Enter”.',
-      terminal_challenge_lines: `Enter your commands here and press Enter...\n The variable $scriptSigHex is already defined for you.\n\n var $scriptSigHex = '04fff...e6b73'`,
+      terminal_challenge_lines: 'Enter your commands here and press Enter...\n The variable $scriptSigHex is already defined for you.\n\n var $scriptSigHex = '04fff...e6b73'',
```

Comment:
The backtices were there for a reason since this doesn't compile. Please run `yarn build`, it should fail with this change and give you a sense of where you need to revert some changes.